.changeset/proud-adults-bet.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-components': minor
+---
+
+Added a new component-names.json file, which contains a list of all components the package is outputting.
```

Comment:
Can this file be exported from all component packages?
That way you could access it from `components-angular` and `components-react` without having to add `components` as a dev dependency.

Reply:
Not really, I just asked because I came across the same issue when trying to use the "not defined" styles. Since the file is not copied over, I had to add `components` as a dependency on top of the `components-angular`. 

It would be convenient to have a generic solution to copy additional files in all component packages but I agree here it's not mandatory. We can do it as an improvement in another ticket.

Reply:
These specific styles should be implemented in the post-components.css file which is indeed copied over to components-angular and components-react and also ends up in the released package. Did you see that?

Reply:
Ah ok, I didn't see that!